plugins/.gitignore

```diff
@@ -0,0 +1,13 @@
+/job-scheduler/
```

Comment:
Should we call this file `opensearch-plugins/.meta`? Just like dashboards file?

Reply:
So either we include `opensearch` or we don't.

a) `plugins/.meta` and `dashboards-plugins/.meta` (here)
b) `opensearch-plugins/.meta` and `opensearch-dashboards-plugins/.meta`

Since this repo is already `opensearch-plugins` I thought that would be redundant.

Reply:
WDYT @saratvemulapalli ?

Reply:
That makes sense to me. Especially since the repo itself is called `opensearch-plugins`.
Lets exclude `opensearch`.